src/crsAdapter/CetsAdapter.js

```diff
@@ -5,47 +5,80 @@ import moment from 'moment';
 import {SERVICE_TYPES} from '../UbpCrsAdapter';
 
 const CONFIG = {
-    crsDateFormat: 'DDMMYYYY',
-    crsTimeFormat: 'HHmm',
+    crs: {
+        crsDateFormat: 'DDMMYYYY',
```

Comment:
I'm undecided whether keeping the `crs` prefix here helps readability or whether it's mostly redundant.

Reply:
you are right, that is redundant. i will remove the the "crs" prefix.